<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWindow</string>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks|QMainWindow::ForceTabbedDocks|QMainWindow::VerticalTabs</set>
  </property>
  <widget class="QWidget" name="centralwidget" />
  <widget class="QMenuBar" name="menuBar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1038</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRun" >
    <property name="title" >
     <string>Run</string>
    </property>
    <addaction name="actionLoad_target" />
    <addaction name="actionRun" />
    <addaction name="actionContinue" />
    <addaction name="actionStep_into" />
    <addaction name="actionStep_over" />
    <addaction name="actionStep_out" />
    <addaction name="actionPause" />
    <addaction name="actionKill" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen_file" />
    <addaction name="actionClose_file" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuRun" />
  </widget>
  <widget class="QStatusBar" name="statusBar" />
  <widget class="QDockWidget" name="dwLog" >
   <property name="windowTitle" >
    <string>Log</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents" >
    <layout class="QVBoxLayout" name="verticalLayout" >
     <item>
      <widget class="QPlainTextEdit" name="pteLog" >
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="tabStopWidth" >
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwConsole" >
   <property name="windowTitle" >
    <string>Console</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2" >
    <layout class="QVBoxLayout" name="verticalLayout_2" >
     <item>
      <widget class="QPlainTextEdit" name="pteConsole" >
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="tabStopWidth" >
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dwPipe" >
   <property name="windowTitle" >
    <string>Pipe</string>
   </property>
   <attribute name="dockWidgetArea" >
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3" >
    <layout class="QVBoxLayout" name="verticalLayout_3" >
     <item>
      <widget class="QPlainTextEdit" name="ptePipe" >
       <property name="tabChangesFocus" >
        <bool>true</bool>
       </property>
       <property name="undoRedoEnabled" >
        <bool>false</bool>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
       <property name="tabStopWidth" >
        <number>30</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_file" />
   <addaction name="actionClose_file" />
   <addaction name="separator" />
   <addaction name="actionLoad_target" />
   <addaction name="separator" />
   <addaction name="actionRun" />
   <addaction name="actionContinue" />
   <addaction name="actionStep_into" />
   <addaction name="actionStep_over" />
   <addaction name="actionStep_out" />
   <addaction name="separator" />
   <addaction name="actionPause" />
   <addaction name="actionKill" />
  </widget>
  <action name="actionRun" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/run.png</normaloff>:/icons/run.png</iconset>
   </property>
   <property name="text" >
    <string>Run</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionContinue" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/play.png</normaloff>:/icons/play.png</iconset>
   </property>
   <property name="text" >
    <string>Continue</string>
   </property>
   <property name="shortcut" >
    <string>F4</string>
   </property>
  </action>
  <action name="actionStep_into" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/step-into.png</normaloff>:/icons/step-into.png</iconset>
   </property>
   <property name="text" >
    <string>Step into</string>
   </property>
   <property name="shortcut" >
    <string>F5</string>
   </property>
  </action>
  <action name="actionStep_over" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/step-next.png</normaloff>:/icons/step-next.png</iconset>
   </property>
   <property name="text" >
    <string>Step over</string>
   </property>
   <property name="shortcut" >
    <string>F6</string>
   </property>
  </action>
  <action name="actionPause" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/stop.png</normaloff>:/icons/stop.png</iconset>
   </property>
   <property name="text" >
    <string>Pause</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionLoad_target" >
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/exec-open.png</normaloff>:/icons/exec-open.png</iconset>
   </property>
   <property name="text" >
    <string>Load target</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionStep_out" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/step-out.png</normaloff>:/icons/step-out.png</iconset>
   </property>
   <property name="text" >
    <string>Step out</string>
   </property>
   <property name="shortcut" >
    <string>F7</string>
   </property>
  </action>
  <action name="actionKill" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/kill.png</normaloff>:/icons/kill.png</iconset>
   </property>
   <property name="text" >
    <string>Kill</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+K</string>
   </property>
  </action>
  <action name="actionOpen_file" >
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/file-open.png</normaloff>:/icons/file-open.png</iconset>
   </property>
   <property name="text" >
    <string>Open</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose_file" >
   <property name="icon" >
    <iconset resource="resources/monkeydbg.qrc" >
     <normaloff>:/icons/file-close.png</normaloff>:/icons/file-close.png</iconset>
   </property>
   <property name="text" >
    <string>Close</string>
   </property>
   <property name="shortcut" >
    <string>Ctrl+W</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/monkeydbg.qrc" />
 </resources>
 <connections/>
</ui>
